protocol-designer/src/pages/Designer/ProtocolSteps/Timeline/ConnectedStepInfo.tsx

```diff
@@ -48,6 +50,10 @@ export function ConnectedStepInfo(props: ConnectedStepInfoProps): JSX.Element {
   const hasFormLevelWarningsPerStep = useSelector(
     dismissSelectors.getHasFormLevelWarningsPerStep
   )
+  const stepListAfterError =
```

Comment:
since the return is an array

```suggestion
  const stepListAfterErrors =
```